libs/tools/barista/src/utils/slugify.ts

```diff
@@ -15,11 +15,14 @@
  */
 
 export function slugify(str: string): string {
```

Comment:
In the case that str is null - you return null now - please specify this in the return value
```suggestion
export function slugify(str: string): string | null {
```